src/components/tools/toolbar/Toolbar.js

```diff
@@ -232,6 +235,11 @@ export default function ToolsToolbar(props) {
                     </Button>
                 </DialogActions>
             </Dialog>
+            <NewToolRequestDialog
+                open={requestDialogOpen}
+                onClose={() => setRequestDialogOpen(false)}
+                baseId={baseId}
```

Comment:
Minor, but this `baseId` shouldn't be necessary.  Dialogs start with their own ID since they're not children to something else.